lib/hypgen/workflow.rb

```diff
@@ -53,6 +53,8 @@ def run!
         output = pipe.read
 
         puts "Workflow execution output: #{output}"
+        pipe.close
+        $?.to_i
```

Comment:
@mkasztelnik yeah, my bad:( too much ruby for one day. This .to_is is converting a 'result object' to simple return code, so it's necessary there.